### PR TITLE
chore: update timeout to 30 sec, add error to slack msgs

### PR DESCRIPTION
**Pull request by jhockett**
*Description of changes:*
- update npm-token-rotation timeout to 30 seconds for token creation
- add error to slack messages on failure
- add alarm description

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class NpmTokenRotationStack extends BaseStack {
     );
 
     const deleteOldTokenStateMachine = this.buildTokenDeletionStateMachine(
-      core.Duration.minutes(15),
```
Comment (johnpc) [inline src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts:65]: Nice! Was the delete token step the one that was timing out?

**Review by johnpc — APPROVED** [review]

**Comment by edwardfoyle** [comment]
Isn't it the create-new-token lambda that was timing out? https://github.com/aws-amplify/amplify-ci-support/blob/main/src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts#L26
Looks like it's using the default timeout of 3 secs as documented here https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-lambda.Function.html#timeout

File: src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts (hunk 1 above)
Comment (jhockett) [reply src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts]: Nope, that was me being confused 😅

**Review by edwardfoyle — APPROVED** [review]

**Review by awsluja — APPROVED** [review]

**Comment by edwardfoyle** [comment]
If it's not too difficult, could we update the alarm message to point to a runbook? Looks like the alarms are created here: https://github.com/aws-amplify/amplify-ci-support/blob/53b781ee6e33c00f8bde010eee0e8c7314d7c411/src/credentials_rotators/npm-token-rotation/src/stacks/base-stack.ts#L106

**Comment by jhockett** [comment]
> If it's not too difficult, could we update the alarm message to point to a runbook? Looks like the alarms are created here:
> 
> https://github.com/aws-amplify/amplify-ci-support/blob/53b781ee6e33c00f8bde010eee0e8c7314d7c411/src/credentials_rotators/npm-token-rotation/src/stacks/base-stack.ts#L106

Yeah, I planned on opening a separate PR for that.

**Review by edwardfoyle — APPROVED** [review]

**Review by awsluja — APPROVED** [review]

**Review by awsluja — APPROVED** [review]

**Review by royjit — APPROVED** [review]

**Pull request merged**